src/main/webapp/styles/style.css

```diff
@@ -161,3 +161,9 @@ body {
   width: 100%;
```

Comment:
small nit, if you get the chance add a modifier that makes sure that the height and width (i'm assuming they are the same to keep it a square) is no larger than the available screen space. On my screen the image is taller than the monitor and I have scroll down to see the bottom groups.

Reply:
Yeah that's a good idea, I'll probably change it so that it takes the minimum of either window.innerWidth and window.innerHeight since I think it has to be a square shaped container.